build/webpack.base.config.js

```diff
@@ -71,26 +66,12 @@ module.exports = {
     logLevel,
     getBaseConfig(config) {
         return {
-            mode: environment,
-            devtool: isProd ? 'source-map' : 'eval-source-map',
```

Comment:
Moved this to the client build so we don't have to turn it off for server builds